manim/utils/cfg_file_utils.py

```diff
@@ -86,31 +87,71 @@ def main():
-    if save_to_userpath.lower() == "y":
+    if save_to_userpath.lower() == "y" or level=="user":
         if not os.path.exists(os.path.abspath(os.path.join(config_paths[1], ".."))):
             os.makedirs(os.path.abspath(os.path.join(config_paths[1], "..")))
         with open(config_paths[1], "w") as fp:
             config.write(fp)
         console.print(
-            f"""A configuration file called [yellow]{config_paths[1]}[/yellow] has been created with your required changes.
+            f"""A configuration file at [yellow]{config_paths[1]}[/yellow] has been created with your required changes.
 This will be used when running the manim command. If you want to override this config,
 you will have to create a manim.cfg in the local directory, where you want those changes to be overridden."""
         )
     else:
         with open(config_paths[2], "w") as fp:
             config.write(fp)
         console.print(
-            f"""A configuration file called [yellow]{config_paths[2]}[/yellow] has been created.
+            f"""A configuration file at [yellow]{config_paths[2]}[/yellow] has been created.
 To save your theme please save that file and place it in your current working directory, from where you run the manim command."""
         )
 
+def show():
+    current_config = curr_config_dict()
+    for category in current_config:
+        console.print(f"{category}",style="bold green underline")
+        for entry in current_config[category]:
+            if category=="logger":
+                console.print(f"{entry} :",end="")
+                console.print(
+                    f" {current_config[category][entry]}",
+                    style=current_config[category][entry]
+                    )
+            else:
+                console.print(f"{entry} : {current_config[category][entry]}")
+        console.print("\n")
 
-if __name__ == "__main__":
-    main()
+def export(path):
+    config = _run_config()[1]
+    if os.path.abspath(path) == os.path.abspath(os.getcwd()):
+        console.print(
+            """You are reading the config from the same directory you are exporting to.
+This means that the exported config will overwrite the config for this directory.
+Are you sure you want to continue?[y/n]""",
+            style="red bold", end=""
+            )
+        proceed = True if input().lower()=="y" else False
+    else:
+        proceed = True
+    if proceed:
+        if not os.path.isdir(path):
+            console.print(f"Creating folder: {path}.",style="red bold")
+            os.mkdir(path)
+        with open(os.path.join(path,"manim.cfg"),"w") as outpath:
+            config.write(outpath)
+            from_path = os.path.join(os.getcwd(),'manim.cfg')
+            to_path = os.path.join(path,'manim.cfg')
+        console.print(f"Exported final Config at {from_path} to {to_path}.")
+    else:
+        console.print("Could NOT write config.", style="red bold")
```

Comment:
Question: should this be an error instead of a console.print?

Reply:
I don't think this qualifies as an "error", per se, since it only crops up in the event that the end user explicitly states that they don't want to perform the action.

_Also I didn't want to import logger just for this, since it's only going to be used in one place :)_